homeassistant/components/shelly/__init__.py

```diff
@@ -766,15 +766,12 @@ async def async_trigger_ota_update(self, beta: bool = False) -> None:
             self.device.firmware_version,
             new_version,
         )
-        result = None
         try:
             async with async_timeout.timeout(AIOSHELLY_DEVICE_TIMEOUT_SEC):
-                result = await self.device.trigger_ota_update(beta=beta)
+                await self.device.trigger_ota_update(beta=beta)
         except (asyncio.TimeoutError, OSError) as err:
             LOGGER.exception("Error while perform ota update: %s", err)
 
-        LOGGER.debug("Result of OTA update call: %s", result)
```

Comment:
I think that this was left for a purpose, it is not unused when debug is enabled, @mib1185 ?

Reply:
I've added the debug logging back. However, `result` will always be `None`.

```py
    async def trigger_ota_update(self, beta: bool = False) -> None:
        """Trigger an ota update."""
        params = {"stage": "beta"} if beta else {"stage": "stable"}
        await self._wsrpc.call("Shelly.Update", params)
```
https://github.com/home-assistant-libs/aioshelly/blob/main/aioshelly/rpc_device.py#L123-L126